### ISSUE TITLE
Add new module about maxima detection

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#125]
@manerotoni 
There seems to be a lot to learn here (at least for me)
See the discussions here:
- https://github.com/scikit-image/scikit-image/issues/3016
- https://forum.image.sc/t/new-maxima-finder-menu-in-fiji/25504/5

I found the concept of "prominence" always confusing, but I think this is super important!
Making a teaching module would be a good way to obtain solid understanding ;-)

**Comment by manerotoni** [comment]
Yes very good point. It is a useful feature and often I realize that I am not sure what it is exactly doing (depending on the implementation). I just use it.
